### ISSUE TITLE
could not load and convert sciex wiff format

### ISSUE THREAD

**Issue by amenglar** [OpenChrom/openchrom#452]
hi, I'm trying to find a tool on Linux OS to convert vendor ms files (including sciex, waters, etc) to mzml format. the most popular solution ms-convert is only available on windows. the openchrom with the ability to convert and run on linux is a good choice.

however when I firstly tested a chromatogram sample of sciex wiff format, it was nothing loaded. 
I also tested this file with msconvert (proteowizard), I could successfully load the peaks.

<img width="585" alt="image" src="https://github.com/OpenChrom/openchrom/assets/7029125/2feda8c8-2468-4d0c-be92-940af69b5ecd">

[datafiles.zip](https://github.com/OpenChrom/openchrom/files/14133418/datafiles.zip)

**Comment by eselmeister** [comment]
@amenglar Thanks for the demo file. We will put it on our todo list.

**Comment by amenglar** [comment]
Another problem, I couldn't download of the older version, 1.4.x by either way. I don't know if I can post it here.

<img width="761" alt="image" src="https://github.com/OpenChrom/openchrom/assets/7029125/699f05e5-8989-4802-9784-e70b5e730e32">

**Comment by eselmeister** [comment]
Yep, no worries. We recommend to use the latest 1.5 rolling release. Implementing the *.wiff converter is on our todo list.

**Comment by ricoderks** [comment]
@amenglar Have a look at [this](https://hub.docker.com/r/chambm/pwiz-skyline-i-agree-to-the-vendor-licenses) page. Here msconvert runs in a docker container. I've been using this a lot to convert to mzxml / mzml on a Linux machine.

**Comment by Ramoonus** [comment]
related to #82
